src/chips/sam4l/spi_dma.rs

```diff
@@ -0,0 +1,383 @@
+use helpers::*;
```

Comment:
Looks like this file should superceded `spi.rs` completely. Can we just rename this file on top of that one? That should also preserve history better in git for code that was copy/pasted between the two files.